tests/integration/dbclient/jdbc/src/test/java/io/helidon/tests/integration/dbclient/jdbc/init/CheckIT.java

```diff
@@ -75,6 +76,11 @@ public void accept(Config config) {
                     if (System.currentTimeMillis() > endTm) {
                         return;
                     }
+                    try {
+                        Thread.sleep(1000);
+                    } catch (InterruptedException ie) {
+                        System.out.println("STATUS: " + ie.getMessage());
```

Comment:
This is waiting for the database to come up. It takes quite a long time so second is fine. I can lower it to 250ms but it will just spam log much more.
System.out.println will be removed. It's debugging output that I forgot to delete.

Reply:
Sleep time reduced to 250ms. No System.out remained in the code.